File: BOJ/5000 ~ 6000/P5597.java
// https://www.acmicpc.net/problem/5597

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class P5597 {
    public static void main(String[] args) throws IOException {
       int[] arr = new int[31];
        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);

       for(int i=1; i<=28; i++) {
           int studentNo = Integer.parseInt(br.readLine());
           arr[studentNo] = 1;
       }

       for (int i=1; i<=30; i++) {
           if (arr[i] == 0) {
               System.out.println(i);
           }
       }
    }
}

File: BOJ/5000 ~ 6000/P5597.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main(args: Array<String>) = with(
    BufferedReader(InputStreamReader(System.`in`))
) {
    // https://kotlinlang.org/docs/arrays.html#create-arrays
    val initArray = Array<Int>(31) { 0 }

    for (idx in 1..28) {
        val studentNo = readLine().toInt()
        initArray[studentNo] = 1
    }

    for (idx in 1..30) {
        if (initArray[idx] == 0)
            println(idx)
    }
}
